ruled_labels/specs_polkadot.yaml

```diff
@@ -0,0 +1,250 @@
+    color: 0C7BAD
+  - name: J2-unconfirmed
+    description: "Issue might be valid, but it's not yet known."
+    color: "000000"
+  - name: J3-intended
+    description: Issue describes a behavior which turns out to work as intended. Closer should explain why.
+    color: b10d11
+  - name: J4-duplicate
+    description: Issue is a duplicate. Closer should comment with a link to the duplicate.
+    color: D62209
+  - name: J5-wont_fix
+    description: "Issue is in principle valid, but this project will not address it. Closer should explain why."
+    color: FB3701
+  - name: J6-invalid
+    description: Issue is invalid. Closer should comment why.
+    color: F76E02
+  - name: S0-design
+    description: PR/Issue is in the design stage
+    color: dd6e83
+  - name: S1-implement
+    description: PR/Issue is in the implementation stage
+    color: c9d7ff
+  - name: S2-test/monitor
+    description: PR/Issue is in the testing stage.
+    color: dbea04
+  - name: S3-deploy
+    description: PR/Issue is in the deployment stage
+    color: 34C302
+  - name: S4-blocked
+    description: "PR/Issue is blocked, see comments for further information."
+    color: D5C978
+  - name: S5-on_ice
+    description: "Work is currently on hold, see comments for further information."
+    color: "122381"
+  - name: T0-node
+    description: This PR/Issue is related to the topic “node”.
+    color: fbffe0
+  - name: T1-runtime
+    description: This PR/Issue is related to the topic “runtime”.
+    color: F5FCE6
+  - name: T2-API
+    description: This PR/Issue is related to APIs.
+    color: 009B40
+  - name: T3-relay_chain
+    description: This PR/Issue is related to the relay chain.
+    color: ffeeee
+  - name: T4-parachains_engineering
+    description: "This PR/Issue is related to Parachains performance, stability, maintenance."
+    color: 42F808
+  - name: T5-parachains_protocol
+    description: This PR/Issue is related to Parachains features and protocol changes.
+    color: 14C663
+  - name: T6-XCM
+    description: This PR/Issue is related to XCM.
+    color: c5def5
+  - name: T7-substrate
+    description: This is an issue that needs to be implemented upstream in Substrate.
+    color: 1d76db
+  - name: T8-release
+    description: This PR/Issue is related to topics touching the release notes.
+    color: 0052cc
+  - name: U0-drop_everything
+    description: Everyone should address the issue now.
+    color: 32024c
+  - name: U1-asap
+    description: "No need to stop dead in your tracks, however issue should be addressed as soon as possible."
+    color: 470c68
+  - name: U2-some_time_soon
+    description: Issue is worth doing soon.
+    color: "673980"
+  - name: U3-nice_to_have
+    description: Issue is worth doing eventually.
+    color: 856d92
+  - name: U4-some_day_maybe
+    description: Issue might be worth doing eventually.
+    color: d2d1d3
+  - name: Z0-unconfirmed
```

Comment:
duplicate of J2.

Reply:
will be deleted after we changed the bot (see "After merge to-dos")